platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_general.adoc

```diff
@@ -285,3 +284,51 @@ Now it can be executed using only `gradle run`, and the continue property will a
 === Tags
 
 <<tags_reference.adoc#tag:properties,`#properties`>>
+
+
+[[name_your_root_project]]
+== Name Your Root Project
+
+Always name your root project in the `settings.gradle(.kts)` file.
+
+=== Explanation
+
+While an empty `settings.gradle(.kts)` file is all that is minimally necessary for a multi-project build, you should always set the `rootProject.name` property in it.
+
+Without setting a name explicitly, the root project's name implicitly defaults to the name of the directory containing the build.
+Some directory names may not be valid or suitable as project names, such as those containing spaces, Gradle logical path separators or other special characters.
+The paths to tasks in build should be able to be specified reliably regardless of where the build lives.
+
+Explicitly setting the root project's name also ensures that it referred to consistently.
```

Comment:
```suggestion
Explicitly setting the root project's name also ensures that it is referred to consistently.
```
Or even rewrite for a simpler sentence 😅

Reply:
I'll wait for Laura for that - she's usually good at rewriting my too complex sentences.